bookmarks/views.py

```diff
@@ -83,6 +84,18 @@ def delete_bookmark_category(request, category_id):
         return HttpResponseRedirect(next)
     else:
         return HttpResponseRedirect(reverse("bookmarks-for-user", args=[request.user.username]))
+    
+@transaction.atomic()
+def download_bookmark_category(request, category_id):
+    category = get_object_or_404(BookmarkCategory, id=category_id)
+    licenses_url = (reverse('bookmark-category-licenses', args=[category_id]))
+    #missing: cache checking done in packdownload
```

Comment:
I would suggest to improve this comment, something like: # NOTE: unlike pack downloads, here we are not doing any cache check to... 

Make it clear, so if we look at it in the future we understand it easily.